CHANGELOG.md

```diff
@@ -35,6 +35,18 @@ As of v3.0.0 this project adheres to [Semantic Versioning](http://semver.org/).
   - `lowchoralsignshift` is now `choralsigndownshift`.
   - `highchoralsignshift` is now `choralsignupshift` and its sign is now inverted.
 - `\grecoloredlines` now takes a single argument, a named color, instead of the three components of an RGB color.  As a result, `\redlines` can now use `gregoriocolor`, making the red staff lines consistent with the text, even when the user teaks `gregoriocolor`.  Addresses [#21787 on the old tracker](https://gna.org/bugs/index.php?21787).
+- All formats can now be changed via the `\grechangeformat` command.  This command takes two or three arguments.  The first argument, required, is the format to be altered.  The defined formats are:
```

Comment:
Reading it again, I think the entry in the Changelog should have a length similar to the others, with a pointer to the documentation... The changelog for v 4.0 is already very big, let's not make it unreadable...